docker/clightning/Dockerfile.arm64

```diff
@@ -0,0 +1,220 @@
+#########################
+# Polar needs c-lightning compiled with the DEVELOPER=1 flag in order to decrease the
+# normal 30 second bitcoind poll interval using the argument --dev-bitcoind-poll=<seconds>.
+# When running in regtest, we want to be able to mine blocks and confirm transactions instantly.
+# Original Source: https://github.com/ElementsProject/lightning/blob/v23.02.2/contrib/docker/linuxarm64v8.Dockerfile
+#########################
+
+#########################
+# BEGIN ElementsProject/lightning/contrib/docker/linuxarm64v8.Dockerfile
+#########################
+
+# This dockerfile is meant to compile a core-lightning x64 image
+# It is using multi stage build:
+# * downloader: Download litecoin/bitcoin and qemu binaries needed for core-lightning
+# * builder: Compile core-lightning dependencies, then core-lightning itself with static linking
+# * final: Copy the binaries required at runtime
+# The resulting image uploaded to dockerhub will only contain what is needed for runtime.
+# From the root of the repository, run "docker build -t yourimage:yourtag -f contrib/linuxarm64v8.Dockerfile ."
```

Comment:
Hate to be nitpicky, but there's still a diff in these comments. Can you paste in these lines which I copied from the [linuxarm64v8.Dockerfile](https://github.com/ElementsProject/lightning/blob/v23.02.2/contrib/docker/linuxarm64v8.Dockerfile) file.

```
# This dockerfile is meant to cross compile with a x64 machine for a arm64v8 host
# It is using multi stage build: 
# * downloader: Download litecoin/bitcoin and qemu binaries needed for Core Lightning
# * builder: Cross compile Core Lightning dependencies, then Core Lightning itself with static linking
# * final: Copy the binaries required at runtime
# The resulting image uploaded to dockerhub will only contain what is needed for runtime.
# From the root of the repository, run "docker build -t yourimage:yourtag -f contrib/linuxarm64v8.Dockerfile ."
```